Project-II-Calculator/calculator/src/App.js

```diff
@@ -6,7 +6,7 @@ import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay'
 
 const App = () => {
   return (
-    <div clasName="calculator">
+    <div className="calculator">
       <CalculatorDisplay text="0" />
       <div className="button-container">
         <ActionButton text="clear" />
```

Comment:
**Day 2 Review**

- Great job on building the interface for calculator Aaron. You manage to implement the provided design.
- For future improvements try and use  .map() to generate new components and pass props to them, for creating  reusable components.
- Overall great job 👍